ui/app/src/components/Controls/SearchBar.tsx

```diff
@@ -29,7 +29,6 @@ const useStyles = makeStyles((theme: Theme) =>
     search: {
       position: 'relative',
       background: (props: any) => props.background ?? theme.palette.background.default,
-      width: '100%',
```

Comment:
This component is used in many different places. Did you test ALL of its usages to check that it didn't break elsewhere?

Reply:
All good, tested in all used places